workflow/apm/apm-overview.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: APM Dashboard - Overview
+sidebarTitle: Overview
+---
+
+The Application Performance Monitoring (APM) dahshboard is a unified dashboard that provides real-time insights into the performance of your applications. Whether instrumenting your APMs with the Infrastructure (Infra) Agent or Serverless, all of your APM data is displayed within this section of the Middleware.
+
+# Infra Agent vs Serverless
+
+There are two different ways to instrument your APMs, through the [Infrastructure Agent]() or a Serverless. The Infra Agent uses OpenTelemetry to collect metrics and traces from your application and send them to Middleware. Instrumenting the APM Agent with the Serverless Agent provides the same features as the Infra Agent apart from the Continuous Profiling feature and slight efficiency degradation. The Infra Agent collects continuous profiling data through Pyroscope, which is not made available when Serverless.

```

Comment:
"or sending the data directly to Middleware as a Serverless application. (new line before the next sentence)"

Reply:
Let's also rephrase the OpenTelemetry piece to "The Middleware Agent is an expansion of the OpenTelemetry collector, which comes with optimized compression and batching logic as well as continuous profiling support." 

Then a new line and then an edit on the serverless piece

"Instrumenting the APM in a Serverless fashion results in the same user experience inside of Middleware, apart from Continuous Profiling."

Then remove the last piece about Pyroscope.